File: programmers/lv0/코딩기초트레이닝/연산과조건문/n의배수/Solution.kt
package lv0.코딩기초트레이닝.연산과조건문.n의배수

//fun solution(num: Int, n: Int): Int {
//    return if (num % n == 0) 1 else 0
//}
fun solution(num: Int, n: Int) = if (num % n == 0) 1 else 0

fun main() {
    println(solution(98, 2))
}

File: programmers/lv0/코딩기초트레이닝/연산과조건문/n의배수/Solution.cpp
#include <iostream>

using namespace std;

int solution(int num, int n) {
    return num%n == 0 ? 1 : 0;
}

int main(int argc, char *argv[]) {
    cout << solution(98, 2) << endl;
    cout << solution(34, 3) << endl;
}
